pyrobosim/pyrobosim/gui/main.py

```diff
@@ -12,44 +12,35 @@
 from matplotlib.backends.qt_compat import QtCore
 from matplotlib.backends.backend_qt5agg import NavigationToolbar2QT
 
-from .options import WorldCanvasOptions
 from ..core.robot import Robot
 from ..core.world import World
 
 
-def start_gui(world: World, options: WorldCanvasOptions = WorldCanvasOptions()) -> None:
+def start_gui(world: World) -> None:
```

Comment:
the changes here are undoing some recent improvements -- please rebase on `main`